moveit_simple/include/moveit_simple/moveit_simple.h

```diff
@@ -121,10 +124,9 @@ class Robot
    * @param frame - frame (must be a TF accessible frame) in which pose is defined
    * @param time - time from start of trajectory to reach point
    * @param point_name - (optional) name of point (used in log messages)
-   *
-   * @return true if point successfullly added
+   * \throws <tf2::TransformException> (Transform from frame to robot base failed)
```

Comment:
Does `@throws` works too?  If so, let's stick with the convention of this file (here and comments below).